core/dbt/graph/selector_methods.py

```diff
@@ -443,8 +468,13 @@ def search(
             )
 
         state_checks = {
-            'modified': self.check_modified,
             'new': self.check_new,
```

Comment:
```suggestion
            # maybe with some sort of comment here.
            'new': old is None,
```

Reply:
I think this needs to be a function/lambda still. maybe?
```suggestion
            'new': lambda old, _: old is None,
```

Reply:
ah yeah that might be right.

Reply:
lambda's a neat idea. I'll leave a comment, too